deploy/stacks/lambda_api.py

```diff
@@ -69,24 +69,12 @@ def __init__(
         self.esproxy_dlq = self.set_dlq(f'{resource_prefix}-{envname}-esproxy-dlq')
         esproxy_sg = self.create_lambda_sgs(envname, 'esproxy', resource_prefix, vpc)
 
-        esproxy_loggroup = logs.LogGroup.from_log_group_name(
-            self, 'esproxyloggroup', f'/aws/lambda/{resource_prefix}-{envname}-esproxy'
-        )
-        graphql_loggroup = logs.LogGroup.from_log_group_name(
-            self, 'graphqlloggroup', f'/aws/lambda/{resource_prefix}-{envname}-graphql'
-        )
-        awsworker_loggroup = logs.LogGroup.from_log_group_name(
-            self, 'awsworkerloggroup', f'/aws/lambda/{resource_prefix}-{envname}-awsworker'
-        )
-
         self.elasticsearch_proxy_handler = _lambda.DockerImageFunction(
             self,
             'ElasticSearchProxyHandler',
             function_name=f'{resource_prefix}-{envname}-esproxy',
-            log_group=esproxy_loggroup
-            if esproxy_loggroup is not None
-            else logs.LogGroup(
-                self, 'esproxyloggroup', log_group_name=f'/aws/lambda/{resource_prefix}-{envname}-esproxy'
+            log_group=logs.LogGroup(
+                self, 'esproxyloggroup', log_group_name=f'/aws/lambda/{resource_prefix}-{envname}-backend-esproxy'
```

Comment:
We discussed offline that just by renaming we will lose the historical metrics? are we ok with that?

Reply:
That is what we need to decide, I have been looking for ways of migrating logs from one log group to another and it is not completely straightforward